shared/src/types/Post.ts

```diff
@@ -0,0 +1,24 @@
+import type {Timestamp} from 'firebase-admin/firestore';
+import {LANGUAGE_TAG} from '../constants/i18n';
+import {UserProfile} from './User';
+
+export type PostFields = {
+  id: string;
+  exerciseId: string;
+  userId?: string;
```

Comment:
I guess this is only used for the anonymity switch and not to keep track of your own sharings?

Reply:
That was the idea yes but I think I didn't follow the thought through 😄